task_processing/plugins/mesos/translator.py

```diff
@@ -2,17 +2,17 @@
 
 # https://github.com/apache/mesos/blob/master/include/mesos/mesos.proto
 MESOS_TASK_STATUS_TO_EVENT = {
-    0: Event(platform_type='starting'),
-    1: Event(platform_type='running'),
-    2: Event(platform_type='finished'),
-    3: Event(platform_type='failed'),
+    0: Event(platform_type='starting', terminal=False),
```

Comment:
should `None` mean anything different from `False` then? if it does, we need to make sure an error is thrown when such event ends up in `if event.terminal` (make an object with __bool__ method that throws an exception). if it doesn't - then let's at least make the field mandatory?